### PR TITLE
SCRUM-1067 mk2

### PR DESCRIPTION
**Pull request by markquintontulloch**
Add fileExtension to BulkLoad entity and select from dropdown in UI on creation of bulk load.

### CODE REVIEW

**Comment by oblodgett** [comment]
@markquintontulloch I think it would be better to not have it be an enum but rather just free text... this way other file types can be added without code changes. (gff, obo, vcf, gaf, etc, etc)

**Comment by markquintontulloch** [comment]
@oblodgett - fair enough, done

File: src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js
Comment (oblodgett) [inline src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js:181]: @markquintontulloch sorry last item... not sure if this field is specified somewhere else? Maybe disableFormFields is just a boolean.

**Review by oblodgett — APPROVED** [review]

File: src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js
Comment (oblodgett) [reply src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js:181]: This is used so that fileExtension can't be changed after files have been uploaded.

Comment (markquintontulloch) [reply src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js:181]: isn't that what we want, otherwise the filename won't match the saved file?

Comment (oblodgett) [reply src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js:181]: Correct... I assume that disableFormFields is a boolean and not a list of fields to disable. If it is a list, then this field would need to be specified in another place to "disable" after editing @adamgibs ? Maybe it's just late and I need to look at this in the morning.

Comment (markquintontulloch) [reply src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js:181]: It does disable the field from being edited on my local system, but I'll let @adamgibs take a look before merging in case I'm doing something funky!

Comment (oblodgett) [reply src/main/cliapp/src/containers/dataLoadsPage/NewBulkLoadForm.js:181]: Oh ok... then yes looks good

**Review by oblodgett — APPROVED** [review]

**Pull request merged**
